datalake/src/main/java/com/sequenceiq/datalake/service/sdx/SdxService.java

```diff
@@ -440,11 +441,14 @@ private void validateRazEnablement(SdxClusterRequest sdxClusterRequest, Detailed
             if (!razEntitlementEnabled) {
                 validationBuilder.error("Provisioning Ranger Raz is not enabled for this account.");
             }
-            if (!AZURE.name().equalsIgnoreCase(environment.getCloudPlatform())) {
-                validationBuilder.error("Provisioning Ranger Raz is only valid for Microsoft Azure.");
+            if (!(AWS.name().equalsIgnoreCase(environment.getCloudPlatform()) || AZURE.name().equalsIgnoreCase(environment.getCloudPlatform()))) {
```

Comment:
We reuse the ignoresCase a lot with treating checks for the cloud platform.  Lets put that in a variable so we can just do an enum comparison.

Reply:
Thanks, addressed.